src/prospectScraper/Program.cs

```diff
@@ -304,44 +305,35 @@ private static void CreateCombinedCSV()
             fileDest.Close();
         }
 
-        private static void CreateCombinedCSVWithExtras()
+        // Takes in CSV file name, class, and class map and returns a list of the class passed in
+        private static List<T> ListMapper<T , TMap>(string csvFileName) where TMap : ClassMap<T>
         {
-            File.AppendAllText($"logs{Path.DirectorySeparatorChar}Status.log", "Creating the big CSV....." + Environment.NewLine);
+            List<T> newListT;
 
-            // Get Schools and the States where they are located.
-            List<School> schoolsAndConferences;
-            using (var reader = new StreamReader($"info{Path.DirectorySeparatorChar}SchoolStatesAndConferences.csv"))
+            using (var reader = new StreamReader($"info{Path.DirectorySeparatorChar}{csvFileName}"))
             using (var csv = new CsvReader(reader, CultureInfo.CurrentCulture))
             {
-                csv.Configuration.RegisterClassMap<SchoolCsvMap>();
-                schoolsAndConferences = csv.GetRecords<School>().ToList();
+                csv.Configuration.RegisterClassMap<TMap>();
+                newListT = csv.GetRecords<T>().ToList();
             }
 
-            List<Region> statesAndRegions;
-            using (var reader = new StreamReader($"info{Path.DirectorySeparatorChar}StatesToRegions.csv"))
-            using (var csv = new CsvReader(reader, CultureInfo.CurrentCulture))
-            {
-                csv.Configuration.RegisterClassMap<RegionCsvMap>();
-                statesAndRegions = csv.GetRecords<Region>().ToList();
-            }
+            return newListT;
+        }
+
+        private static void CreateCombinedCSVWithExtras()
+        {
+            File.AppendAllText($"logs{Path.DirectorySeparatorChar}Status.log", "Creating the big CSV....." + Environment.NewLine);
+
+            // Get Schools and the States where they are located.
+            List<School> schoolsAndConferences = ListMapper<School, SchoolCsvMap>("SchoolStatesAndConferences.csv");
+
+            List<Region> statesAndRegions = ListMapper<Region, RegionCsvMap>("");
```

Comment:
Why does this have an empty string instead of StatesToRegions.csv?

Reply:
That is an error on my part, I have corrected it and added the CSV filename 330.